src/utils/fileLoaders.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+const jsyaml = require('js-yaml');
+
+const cwd = process.cwd();
+
+const file = (refValue, options) => {
```

Comment:
Typical convention is for functions to start with verbs. Currently it is really unclear what it does.

Reply:
this what `json-schema-deref-sync` requires in loaders